index.html

```diff
@@ -556,6 +560,8 @@
           </li>
           <li>Let <var>request</var> be a new <a>PaymentRequest</a>.
           </li>
+          <li>Set <var>request</var>.<a>[[\paymentRequestID]]</a> to <var>paymentRequestID</var>.
```

Comment:
Putting this in payment details will naturally demote the importance of this field. Maybe this should be discussed in the f2f then.